algonaut_client/Cargo.toml

```diff
@@ -27,3 +27,8 @@ dotenv = "0.15.0"
 rand = "0.8.3"
 getrandom = { version = "0.2.2", features = ["js"] }
 tokio = { version = "1.6.0", features = ["macros"] }
+
+[features]
+default = ["native"]
```

Comment:
Why is `native` being defined as default here and defaults disabled [here](https://github.com/manuelmauro/algonaut/blob/55fbbedff1e57d82bf106bafa2d54dd0a085aa89/Cargo.toml#L22)? To use `algonaut_client` in other contexts perhaps?